<?xml version="1.0" ?>
<launch>
  
  <node pkg="nostop_agent" type="Guard" name="$(anon Guard)" required="true" output="screen">
    <param name="robot_name" value="$(optenv ROS_ROBOT_NAME red_blue)" />
    <param name="learning_name" value="$(optenv ROS_LEARNING_ALG DISL)" />
    <param name="period" value="$(optenv ROS_AGENT_PERIOD 1)" />
  </node>

</launch>